custom/aaa/models.py

```diff
@@ -1184,7 +1185,7 @@ def _dictfetchall(cursor):
     "Return all rows from a cursor as a dict"
     columns = [col[0] for col in cursor.description]
     return [
-        dict(zip(columns, row))
+        dict(list(zip(columns, row)))
```

Comment:
The modernize script is adding code here that is not necessary and only makes it harder to read. I'd prefer see us not add stuff like this since I doubt we will ever go back and remove it once we're on Python 3.

Reply:
Good point, updated: https://github.com/dimagi/commcare-hq/pull/23840/commits/576896b5480116756b68c7e1109b64062e84f1fb